install_config/cluster_metrics.adoc

```diff
@@ -12,307 +12,433 @@ toc::[]
+deployment.
 
-----
-$ oadm policy add-cluster-role-to-user \
-    cluster-reader \
-    system:serviceaccount:default:heapster
-----
+However, the only required parameter is `*HAWKULAR_METRICS_HOSTNAME*`. This
+value is required when creating the deployer because it specifies the hostname
+for the Hawkular Metrics
+link:../architecture/core_concepts/routes.html[route]. You will need to know the
+value of `*HAWKULAR_METRICS_HOSTNAME*` when
+link:#accessing-metrics-in-the-console[configuring the console] for metrics
+access.
+
+== Deploying the Metric Components
 
-The following is the definition of the *Heapster* replication controller:
+Since deploying and configuring all the metric components is handled by the
+*metrics deployer*, you can simply deploy everything in one step.
 
+The following examples show you how to deploy metrics with and without
+persistent storage using the default template parameters. Optionally, you can
+specify any of the
+link:../install_config/cluster_metrics.html#deployer-template-parameters[template parameters]
+when calling these commands.
+
+.Deploying with Persistent Storage
 ====
-[source,yaml]
+The following command sets the Hawkular Metrics route to use
+`hawkular-metrics.example.com` and is deployed using persistent storage.
+
+You must have a persistent volume of sufficient size available.
+
+[options="nowrap"]
 ----
-apiVersion: "v1"
-kind: "List"
-items:
-  -
-    apiVersion: "v1"
-    kind: "ReplicationController"
-    metadata:
-      labels:
-        provider: "fabric8"
-        component: "heapster"
-      name: "heapster"
-    spec:
-      replicas: 1
-      selector:
-        provider: "fabric8"
-        component: "heapster"
-      template:
-        metadata:
-          labels:
-            provider: "fabric8"
-            component: "heapster"
-        spec:
-          containers:
-            -
-              args:
-                - "-source=kubernetes:https://kubernetes.default.svc?auth=&insecure=true&useServiceAccount=true"
-                - "-sink=influxdb:http://influxdb-monitoring:8086"
-              image: "kubernetes/heapster:v0.17.0"
-              name: "heapster"
-          serviceAccount: "heapster"
+$ oc process -f metrics.yaml -v \
+HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com,IMAGE_PREFIX=openshift/origin-,IMAGE_VERSION=devel \
+| oc create -f -
 ----
 ====
 
-Save it to a file and create it with:
+.Deploying without Persistent Storage
+====
+The following command sets the Hawkular Metrics route to use
+`hawkular-metrics.example.com` and is deployed without persistent storage.
+Remember, this is being deployed without persistent storage, so metric data loss
+can occur.
 
+[options="nowrap"]
 ----
-$ oc create -f <path/to/heapster.yaml>
+$ oc process -f metrics.yaml -v \
+HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com,IMAGE_PREFIX=openshift/origin-,IMAGE_VERSION=devel,USE_PERSISTENT_STORAGE=false \
```

Comment:
Nope, shouldn't be in there: https://github.com/mwringe/openshift-docs/blob/753365651e1533926a059d8583ad55f7f4dcc438/admin_guide/cluster_metrics.adoc